### ISSUE TITLE
Support converting data back to a canvas

### ISSUE THREAD

**Issue by tiborsaas** [LinusU/resize-image-data#1]
Hello,

I find this little tool really handy. I just wanted to suggest that drawing the resized pixel array back to a canvas is not that trivial so an extra method could help with that. 

If you like, I could send a PR.

**Comment by LinusU** [comment]
Hey, I'm glad you like the library!

I'm actually using this server side and I don't have access to `Canvas`, thus I would prefer to keep those helper methods out of the core lib.

What do you think about creating a new library for drawing ImageData on to a Canvas? Also, have you looked at `putImageData`?

**Comment by tiborsaas** [comment]
It makes sense if you intended it for server use. 

Yes, I used `putImageData`, like this after some stackoverflowing :)

    const resized = resizeImageData(tempData, SAMPLE_W, SAMPLE_H, 'nearest-neighbor');

    let sampleCanvas = document.createElement('canvas');
    let sampleCtx = sampleCanvas.getContext('2d');
    const imageData = sampleCtx.createImageData(SAMPLE_W, SAMPLE_H);
    imageData.data.set(resized.data);

    sampleCtx.putImageData(imageData, 0, 0);

I don't think it would justify a full lib, maybe it could be part of the readme.

**Comment by LinusU** [comment]
I have just pushed and released a new major version (`0.2.0`) of the library, which should make sure that you get proper `ImageData` instances back. That way you shouldn't have to do the `createImageData` + `data.set()` dance anymore ☺️ 

This should allow you to just use `putImageData` without anything else:

```js
const canvas = document.createElement('canvas')
const ctx = canvas.getContext('2d')

// ...

const resized = resizeImageData(tempData, SAMPLE_W, SAMPLE_H, 'nearest-neighbor')
ctx.putImageData(resized, 0, 0)
```

Do you think that this usage is good? 🙌
